Sources/Runestone/TextView/Core/TextView.swift

```diff
@@ -419,6 +419,15 @@ open class TextView: UIScrollView {
             textInputView.gutterTrailingPadding = newValue
         }
     }
+    /// The amout of characters to use for width calculation inside the gutter.
```

Comment:
There's a tiny typo here. Also, would it make sense to write "The minimum amount of characters..." since this isn't necessarily the exact amount that will be used for the width calculation?

```suggestion
    /// The minimum amount of characters to use for width calculation inside the gutter.
```